safe_eth/eth/clients/etherscan_client.py

```diff
@@ -366,6 +366,21 @@ def get_contract_metadata(
                 return ContractMetadata(contract_name, contract_abi, False)
         return None
 
+    def process_response(self, response):
```

Comment:
This name seems a bit generic to me. It does not process all types of Etherescan api responses, it processes a specific type. 

Should it be a private method?